sdk/tables/azure-data-tables/inc/azure/data/tables/tables_clients.hpp

```diff
@@ -391,14 +409,34 @@ namespace Azure { namespace Data { namespace Tables {
         const TableClientOptions& options = {});
 
     /**
-     * @brief List tables.
+     * @brief Create the table indicated in the tableName field of the client.
+     *
+     * @param context for canceling long running operations.
+     * @param tableName The name of the table to be created.
+     * @return Create table result.
+     */
+    Response<Models::Table> CreateTable(std::string tableName, Core::Context const& context = {});
+
+    /**
+     * @brief Delete the table indicated in the tableName field of the client.
+     *
+     * @param context for canceling long running operations.
+     * @param tableName The name of the table to be deleted.
+     * @return Delete table result.
+     */
+    Response<Models::DeleteResult> DeleteTable(
+        std::string tableName,
```

Comment:
Same comment as above: In general, it's better to pass complex objects as `const&` rather than by value, because passing by value requires taking a copy of the object and destroying it afterwards.

The single exception to this is in a method constructor, where there is a school of thought to pass constructor structure parameters which will be used to initialize a struct member by value and then use std::move to move the temporary value to the struct member. 

I'm personally not a fan of this pattern because it pushes the code to construct the copy to the caller (thus every caller has the code to construct the copy) but that isn't relevant - the pattern exists and there are places which recommend it.

Reply:
i missed that, sorry  , my bad, done